scripts/tools/mod_tools.js

```diff
@@ -348,6 +350,38 @@ function hashModName(modName) {
     return hash.substring(0, 16);
 }
 
+// Returns a gulp pattern array, used for matching file types when copying files over from '.temp' folder
+function buildBundleGulp(buildDir, extConfigName){
+    let gulp_array = [
+        buildDir + '/*([0-f])',
```

Comment:
I feel like you should be able to just add another line here like buildDir + '/*([0-f]).stream'  instead of having multiple streams

Reply:
Should be condensed it into a single stream now.